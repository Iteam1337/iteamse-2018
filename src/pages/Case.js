// @flow

import React from 'react'
import GridRow from '../components/Grid/GridRow'
import GridContent from '../components/Grid/GridContent'
import styled from 'styled-components'
import { filterByLocation } from '../utils/filterByLocation'
import FilterByLocation from '../components/FilterByLocation/FilterByLocation'
import Team from '../components/Team/Team'
import Header from '../components/Header/Header'

const CASES = [
  {
    location: 'Stockholm',
    shortDescription:
      'Synliggörande av asylsökandes kompetens med digitala verktyg',
    title: 'Arbetsförmedlingen | Digital tjänst',
  },
  {
    location: 'Göteborg',
    shortDescription: 'API som strategisk framtidssatsning',
    title: 'Taxi Stockholm | API',
  },
  {
    location: 'Stockholm',
    shortDescription: 'Från styrelsebeslut till agil verklighet',
    title: 'SEB | Lorem Ipsum',
  },
  {
    location: 'Stockholm',
    shortDescription: 'En rubrik om vad vi gjort för Vimla',
    title: 'Vimla | Backend',
  },
]

const Cases = styled.div`
  display: grid;
  grid-column-gap: 30px;
  grid-row-gap: 40px;
  grid-template-columns: repeat(2, 1fr);
  padding-bottom: 120px;
  padding-top: 60px;
`
const Case = styled.div``
const CaseImage = styled.div`
  background-color: #f1f1f1;
  height: 500px;
  width: 500px;
`
const Meta = styled.div`
  margin-top: 30px;
`
const Title = styled.div`
  font-size: 18px;
  font-weight: 500;
  margin-bottom: 10px;
`
const ShortDescription = styled.div`
  font-size: 25px;
  font-weight: 300;
`

const CasePage = () => {
  return (
    <React.Fragment>
      <Header
        backgroundImage="http://lorempixel.com/1920/1080"
        messageBgColor="green"
        messageOne="Några exempel på"
        messageTwo="sådant vi gjort"
      />

      <GridRow>
        <GridContent>
          <FilterByLocation>
            {location => (
              <Cases>
                {CASES.filter(filterByLocation(location)).map(workCase => (
                  <Case key={workCase.title}>
                    <CaseImage />
                    <Meta>
                      <Title>{workCase.title}</Title>
                      <ShortDescription>
                        {workCase.shortDescription}
                      </ShortDescription>
                    </Meta>
                  </Case>
                ))}
              </Cases>
            )}
          </FilterByLocation>
        </GridContent>
      </GridRow>

      <Team bgColor="green" shortName={['hrn', 'jmn']} />
    </React.Fragment>
  )
}

export default CasePage
