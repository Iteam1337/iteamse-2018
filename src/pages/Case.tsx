import gql from 'graphql-tag'
import * as React from 'react'
import { Query } from 'react-apollo'
import { Helmet } from 'react-helmet'
import { RouteComponentProps, withRouter } from 'react-router-dom'
import { CasePageQuery, CasePageQueryVariables } from '../../typings/iteamse'
import Block from '../components/Blocks/Block'
import Frameworks from '../components/Blocks/Frameworks'
import Quote from '../components/Blocks/Quote'
import Breadcrumbs from '../components/Breadcrumbs/Breadcrumbs'
import GridColumn from '../components/Grid/GridColumn'
import CaseHeader from '../components/Header/CaseHeader'
import Team from '../components/Team/Team'

export const CASE_PAGE_QUERY = gql`
  query CasePage($slug: String!) {
    workCase: case(slug: $slug) {
      headerBgColor
      logo
      slug
      casePageImage
      casePageBackgroundImage
      frameworks
      frameworksTitle
      tags
      title
      introduction
      introductionTitle
      process
      processTitle
      development
      developmentTitle
      aboutCompany
      aboutCompanyTitle
      partners
      partnersTitle
      contact
      contactTitle
      team
      quote
      quotePerson
    }
  }
`

class CaseQuery extends Query<CasePageQuery, CasePageQueryVariables> {}

const CasePage: React.SFC<RouteComponentProps<{ slug: string }>> = ({
  match,
}) => {
  return (
    <CaseQuery query={CASE_PAGE_QUERY} variables={{ slug: match.params.slug }}>
      {({ loading, data }) => {
        if (loading || !data || !data.workCase) {
          return null
        }

        const { workCase } = data

        return (
          <>
            <Helmet>
              <title>Iteam - There's a better way |Â {workCase.title}</title>
            </Helmet>
            <CaseHeader
              caseImage={workCase.casePageImage}
              caseBackgroundImage={workCase.casePageBackgroundImage}
              logo={workCase.logo}
              tags={workCase.tags}
            />

            <GridColumn>
              <Breadcrumbs title={workCase.title} />

              <Block title={workCase.introductionTitle}>
                {workCase.introduction}
              </Block>

              <Block title={workCase.processTitle}>{workCase.process}</Block>

              {workCase.quote && (
                <Quote person={workCase.quotePerson}>{workCase.quote}</Quote>
              )}

              <Block title={workCase.developmentTitle}>
                {workCase.development}
              </Block>

              <Block title={workCase.aboutCompanyTitle}>
                {workCase.aboutCompany}
              </Block>

              {workCase.partners &&
                workCase.partnersTitle && (
                  <Block title={workCase.partnersTitle}>
                    {workCase.partners}
                  </Block>
                )}

              {workCase.contact &&
                workCase.contactTitle && (
                  <Block title={workCase.contactTitle}>
                    {workCase.contact}
                  </Block>
                )}

              {workCase.frameworks && (
                <Frameworks
                  frameworks={workCase.frameworks}
                  title={workCase.frameworksTitle}
                />
              )}
            </GridColumn>

            <Team bgColor="green" shortName={workCase.team} />
          </>
        )
      }}
    </CaseQuery>
  )
}

export default withRouter(CasePage)
