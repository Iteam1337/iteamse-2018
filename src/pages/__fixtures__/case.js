export const workCase = {
  headerBgColor: '#1616B2',
  logo:
    '//images.ctfassets.net/rj4r6yfcesw5/4p0ptdLj9SaECekOI4c6C2/c2cdeab1cebc9650ebb0e7778357c231/AF_ordbild_vit.png',
  slug: 'arbetsformedlingen',
  thumbnailImage:
    '//images.ctfassets.net/rj4r6yfcesw5/3Y7zZ25WbeOCU8w84kAEos/b97ff1899cf6f2e4dca0eb2b4f0d5a44/af.png',
  tags: ['API', 'Docker', 'UX', 'React', 'Vue'],
  title: 'Arbetsförmedlingen',
  introduction:
    'Aeneanurutrum, erat at tempus porttitor, mi eros vestibulum magna, eget semper mauris ipsum a nisi. Interdum et malesuada fames ac ante ipsum primis in faucibus. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Cras quis odio vel turpis pellentesque finibus. Pellentesque cursus diam ac ligula placerat volutpat. Vivamus dictum augue eu rutrum efficitur. Suspendisse ut pellentesque ante. Suspendisse odio velit, fringilla eu porta vel, elementum ac libero. Nam eu vestibulum magna. Vestibulum quis risus elementum, lobortis sapien at, rhoncus enim. Nam accumsan odio lacus, at facilisis lacus rhoncus vel. Suspendisse sollicitudin a ipsum at cursus.',
  introductionTitle: 'En ny digital tjänst',
  process:
    'Aeneanurutrum, erat at tempus porttitor, mi eros vestibulum magna, eget semper mauris ipsum a nisi. Interdum et malesuada fames ac ante ipsum primis in faucibus. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Cras quis odio vel turpis pellentesque finibus. Pellentesque cursus diam ac ligula placerat volutpat. Vivamus dictum augue eu rutrum efficitur. Suspendisse ut pellentesque ante. Suspendisse odio velit, fringilla eu porta vel, elementum ac libero. Nam eu vestibulum magna. Vestibulum quis risus elementum, lobortis sapien at, rhoncus enim. Nam accumsan odio lacus, at facilisis lacus rhoncus vel. Suspendisse sollicitudin a ipsum at cursus.',
  processTitle: 'Process',
  development:
    'Aeneanurutrum, erat at tempus porttitor, mi eros vestibulum magna, eget semper mauris ipsum a nisi. Interdum et malesuada fames ac ante ipsum primis in faucibus. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Cras quis odio vel turpis pellentesque finibus. Pellentesque cursus diam ac ligula placerat volutpat. Vivamus dictum augue eu rutrum efficitur. Suspendisse ut pellentesque ante. Suspendisse odio velit, fringilla eu porta vel, elementum ac libero. Nam eu vestibulum magna. Vestibulum quis risus elementum, lobortis sapien at, rhoncus enim. Nam accumsan odio lacus, at facilisis lacus rhoncus vel. Suspendisse sollicitudin a ipsum at cursus.',
  developmentTitle: 'Utveckling',
  aboutCompany:
    'Aeneanurutrum, erat at tempus porttitor, mi eros vestibulum magna, eget semper mauris ipsum a nisi. Interdum et malesuada fames ac ante ipsum primis in faucibus. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Cras quis odio vel turpis pellentesque finibus. Pellentesque cursus diam ac ligula placerat volutpat. Vivamus dictum augue eu rutrum efficitur. Suspendisse ut pellentesque ante. Suspendisse odio velit, fringilla eu porta vel, elementum ac libero. Nam eu vestibulum magna. Vestibulum quis risus elementum, lobortis sapien at, rhoncus enim. Nam accumsan odio lacus, at facilisis lacus rhoncus vel. Suspendisse sollicitudin a ipsum at cursus.',
  aboutCompanyTitle: 'Arbetsförmedlingen',
  partners: '[Digitalbyrån](https://hej.com)',
  partnersTitle: 'Samarbetspartners',
  contact:
    '- [Hans Rollman](mailto:hans.rollman@iteam.se)\n- Johan Littorin, Telenor, 0734 41 72 85',
  contactTitle: 'Presskontakt',
  quote:
    '"We feel that our crew at Iteam are more colleagues than suppliers. Rather than specifying a tech spec to work on, we collaborate, carefully chiseling out functionality as we go along. This way of working has minimized waste and it has also taken us further than we ever imagined”',
  quotePerson: 'Anna Andersson, chef Arbetsförmedlingen',
  __typename: 'Case',
}
