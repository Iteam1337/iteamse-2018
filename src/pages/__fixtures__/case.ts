export const workCase = {
  __typename: 'Case',
  aboutCompany:
    'Aeneanurutrum, erat at tempus porttitor, mi eros vestibulum magna, eget semper mauris ipsum a nisi. Interdum et malesuada fames ac ante ipsum primis in faucibus. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Cras quis odio vel turpis pellentesque finibus. Pellentesque cursus diam ac ligula placerat volutpat. Vivamus dictum augue eu rutrum efficitur. Suspendisse ut pellentesque ante. Suspendisse odio velit, fringilla eu porta vel, elementum ac libero. Nam eu vestibulum magna. Vestibulum quis risus elementum, lobortis sapien at, rhoncus enim. Nam accumsan odio lacus, at facilisis lacus rhoncus vel. Suspendisse sollicitudin a ipsum at cursus.',
  aboutCompanyTitle: 'Arbetsförmedlingen',
  contact:
    '- [Hans Rollman](mailto:hans.rollman@iteam.se)\n- Johan Littorin, Telenor, 0734 41 72 85',
  contactTitle: 'Presskontakt',
  development:
    'Aeneanurutrum, erat at tempus porttitor, mi eros vestibulum magna, eget semper mauris ipsum a nisi. Interdum et malesuada fames ac ante ipsum primis in faucibus. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Cras quis odio vel turpis pellentesque finibus. Pellentesque cursus diam ac ligula placerat volutpat. Vivamus dictum augue eu rutrum efficitur. Suspendisse ut pellentesque ante. Suspendisse odio velit, fringilla eu porta vel, elementum ac libero. Nam eu vestibulum magna. Vestibulum quis risus elementum, lobortis sapien at, rhoncus enim. Nam accumsan odio lacus, at facilisis lacus rhoncus vel. Suspendisse sollicitudin a ipsum at cursus.',
  developmentTitle: 'Utveckling',
  headerBgColor: '#1616B2',
  introduction:
    'Aeneanurutrum, erat at tempus porttitor, mi eros vestibulum magna, eget semper mauris ipsum a nisi. Interdum et malesuada fames ac ante ipsum primis in faucibus. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Cras quis odio vel turpis pellentesque finibus. Pellentesque cursus diam ac ligula placerat volutpat. Vivamus dictum augue eu rutrum efficitur. Suspendisse ut pellentesque ante. Suspendisse odio velit, fringilla eu porta vel, elementum ac libero. Nam eu vestibulum magna. Vestibulum quis risus elementum, lobortis sapien at, rhoncus enim. Nam accumsan odio lacus, at facilisis lacus rhoncus vel. Suspendisse sollicitudin a ipsum at cursus.',
  introductionTitle: 'En ny digital tjänst',
  logo:
    '//images.ctfassets.net/rj4r6yfcesw5/4p0ptdLj9SaECekOI4c6C2/c2cdeab1cebc9650ebb0e7778357c231/AF_ordbild_vit.png',
  partners: '[Digitalbyrån](https://hej.com)',
  partnersTitle: 'Samarbetspartners',
  process:
    'Aeneanurutrum, erat at tempus porttitor, mi eros vestibulum magna, eget semper mauris ipsum a nisi. Interdum et malesuada fames ac ante ipsum primis in faucibus. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Cras quis odio vel turpis pellentesque finibus. Pellentesque cursus diam ac ligula placerat volutpat. Vivamus dictum augue eu rutrum efficitur. Suspendisse ut pellentesque ante. Suspendisse odio velit, fringilla eu porta vel, elementum ac libero. Nam eu vestibulum magna. Vestibulum quis risus elementum, lobortis sapien at, rhoncus enim. Nam accumsan odio lacus, at facilisis lacus rhoncus vel. Suspendisse sollicitudin a ipsum at cursus.',
  processTitle: 'Process',
  quote:
    '"We feel that our crew at Iteam are more colleagues than suppliers. Rather than specifying a tech spec to work on, we collaborate, carefully chiseling out functionality as we go along. This way of working has minimized waste and it has also taken us further than we ever imagined”',
  quotePerson: 'Anna Andersson, chef Arbetsförmedlingen',
  slug: 'arbetsformedlingen',
  tags: ['API', 'Docker', 'UX', 'React', 'Vue'],
  team: null,
  thumbnailImage:
    '//images.ctfassets.net/rj4r6yfcesw5/3Y7zZ25WbeOCU8w84kAEos/b97ff1899cf6f2e4dca0eb2b4f0d5a44/af.png',
  title: 'Arbetsförmedlingen',
}
