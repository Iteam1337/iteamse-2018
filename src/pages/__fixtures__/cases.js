export const cases = [
  {
    location: 'Stockholm',
    shortDescription:
      'Synliggörande av asylsökandes kompetens med digitala verktyg',
    slug: 'vimla',
    thumbnailImage:
      '//images.ctfassets.net/rj4r6yfcesw5/68OlBclfQAGMcEUsQgcIUY/9bdf52bb8a9a17be06e11ef3fa32d3de/vimla-case-phone.png',
    title: 'Vimla',
    __typename: 'Case',
  },
  {
    location: 'Stockholm',
    shortDescription:
      'Synliggörande av asylsökandes kompetens med digitala verktyg',
    slug: 'tsab',
    thumbnailImage:
      '//images.ctfassets.net/rj4r6yfcesw5/UGucAeFycSqIGw0CAc0ys/3d58300261bcff64dba751de9228a150/tsab-case-phone.png',
    title: 'Taxi Stockholm',
    __typename: 'Case',
  },
  {
    location: 'Stockholm',
    shortDescription:
      'Synliggörande av asylsökandes kompetens med digitala verktyg',
    slug: 'seb',
    thumbnailImage: null,
    title: 'SEB',
    __typename: 'Case',
  },
  {
    location: 'Stockholm',
    shortDescription:
      'Synliggörande av asylsökandes kompetens med digitala verktyg',
    slug: 'arbetsformedlingen',
    thumbnailImage:
      '//images.ctfassets.net/rj4r6yfcesw5/3Y7zZ25WbeOCU8w84kAEos/b97ff1899cf6f2e4dca0eb2b4f0d5a44/af.png',
    title: 'Arbetsförmedlingen',
    __typename: 'Case',
  },
]
