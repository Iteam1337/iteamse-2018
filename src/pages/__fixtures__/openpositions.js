export const openpositions = [
  {
    id: 'junior-ux-designer',
    location: 'Stockholm',
    role:
      'Som UX designer på Iteam är din roll att hjälpa våra kunder med digitala lösningar som skapar värde på riktigt. Som gör något bra både för kunden, slutanvändaren och samhället. Med hjälp av olika researchmetoder undersöker du både användarens och kundens behov och kontext, samt definierar och testar användarupplevelser.\n\nDu delar med dig av din kunskap inom user experience. Du förstår också att om lösningar ska vara verkligt användarvänliga behöver även arbetsprocesserna och kulturen vara användarcentrerad. Vi strävar mot att UX aldrig ska få vara bara en fas eller en eftertanke. Det ska genomsyra hela projektet. Alltid. ',
    title: 'Junior UX Designer',
    __typename: 'OpenPosition',
  },
]
