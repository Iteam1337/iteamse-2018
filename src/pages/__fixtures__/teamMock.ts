import { TEAM_QUERY } from '../../components/Team/Team'
import { teamMembers } from './teamMembers'

export const TeamQueryMock = {
  request: {
    query: TEAM_QUERY,
  },
  result: {
    data: {
      teamMembers,
    },
  },
}
