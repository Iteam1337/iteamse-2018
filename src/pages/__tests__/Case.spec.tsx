import React from 'react'
import { render, wait } from 'react-testing-library'
import MockedQuery from '../../utils/test-utils/MockedQuery'
import { workCase } from '../__fixtures__/case'
import { TeamQueryMock } from '../__fixtures__/teamMock'
import Case, { CASE_PAGE_QUERY } from '../Case'

const location = {
  pathname: '/case/arbetsformedlingen',
}

const params = {
  slug: 'arbetsformedlingen',
}

const mocks = [
  {
    request: {
      query: CASE_PAGE_QUERY,
      variables: params,
    },
    result: {
      data: {
        workCase,
      },
    },
  },
  TeamQueryMock,
]

describe('components/Case', () => {
  it('renders Case', async () => {
    const { container } = render(
      <MockedQuery location={location} mocks={mocks} params={params}>
        <Case />
      </MockedQuery>
    )

    await wait()

    expect(container).toMatchSnapshot()
  })
})
