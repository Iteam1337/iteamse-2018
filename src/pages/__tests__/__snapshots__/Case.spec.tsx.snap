// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/Case renders Case 1`] = `
<div>
  <div
    data-test="header-case"
    class="GridColumn__GridColumnClean-sc-1xz61ww-0 sc-chPdSV cYkVcK"
  >
    <div
      class="CaseHeader__Content-i46oib-0 dgxEQK"
    >
      <div
        class="Navigation__Wrap-sc-1i56sr4-0 dNCifP"
      >
        <a
          class="sc-csuQGl kBaCvS"
        >
          <img
            data-testid="logo"
            path="test-file-stub"
            class="sc-Rmtcm jeZXze"
          />
        </a>
        <div
          class="Navigation__NavigationItems-sc-1i56sr4-1 kQCxjI"
        >
          <a
            class="sc-bRBYWo cBPUew active-nav"
            aria-current="true"
          >
            Case
          </a>
          <a
            class="sc-bRBYWo cBPUew"
            aria-current="false"
          >
            AI
          </a>
          <a
            class="sc-bRBYWo cBPUew"
            aria-current="false"
          >
            Metod
          </a>
          <a
            class="sc-bRBYWo cBPUew"
            aria-current="false"
          >
            Medarbetare
          </a>
          <a
            class="sc-bRBYWo cBPUew"
            aria-current="false"
          >
            Karriär
          </a>
          <a
            class="sc-bRBYWo cBPUew"
            aria-current="false"
          >
            Om
          </a>
          <div
            class="Navigation__Indicator-sc-1i56sr4-2 gdIgfK"
          />
        </div>
      </div>
      <div
        class="CaseHeader__Information-i46oib-1 fLzChM"
      >
        <div
          class="CaseHeader__Meta-i46oib-2 gbFvKk"
        >
          <img
            alt="Logo"
            src="//images.ctfassets.net/rj4r6yfcesw5/4p0ptdLj9SaECekOI4c6C2/c2cdeab1cebc9650ebb0e7778357c231/AF_ordbild_vit.png"
            class="CaseHeader__Logo-i46oib-4 jTjMzI"
          />
          <div
            class="CaseHeader__Tags-i46oib-5 fKdhSw"
          >
            <div
              class="CaseHeader__Tag-i46oib-6 pwCzF"
            >
              API
            </div>
            <div
              class="CaseHeader__Tag-i46oib-6 pwCzF"
            >
              Docker
            </div>
            <div
              class="CaseHeader__Tag-i46oib-6 pwCzF"
            >
              UX
            </div>
            <div
              class="CaseHeader__Tag-i46oib-6 pwCzF"
            >
              React
            </div>
            <div
              class="CaseHeader__Tag-i46oib-6 pwCzF"
            >
              Vue
            </div>
          </div>
        </div>
        <div
          class="CaseHeader__CaseImage-i46oib-7 cwQkDt"
        >
          <img
            alt=""
            src="//images.ctfassets.net/rj4r6yfcesw5/4cWj2fKl3GOQgSKss0IW0q/0b2322470ffb780b925c8e446e745540/af-cropped-by-aeo.png"
            class="CaseHeader__ProjectImage-i46oib-3 inDYdL"
          />
        </div>
      </div>
    </div>
  </div>
  <div
    class="CaseHeader__CaseImageMobile-i46oib-8 bQjXos"
  >
    <img
      alt=""
      src="//images.ctfassets.net/rj4r6yfcesw5/4cWj2fKl3GOQgSKss0IW0q/0b2322470ffb780b925c8e446e745540/af-cropped-by-aeo.png"
      class="CaseHeader__ProjectImage-i46oib-3 inDYdL"
    />
  </div>
  <div
    class="GridColumn__GridColumnClean-sc-1xz61ww-0 sc-gzVnrw gPskFt"
  >
    <div
      data-test="breadcrumbs"
      class="sc-bdVaJa sc-bZQynM fXeWOs"
    >
      <a>
        <span
          class="Breadcrumbs__StyledSpan-sc-1ajw33n-0 bBBcLz"
        >
          Case
        </span>
      </a>
      <span>
        Arbetsförmedlingen
      </span>
    </div>
    <div
      class="react-reveal"
      style="opacity: 1;"
    >
      <div
        data-test="block-"
        class="sc-bdVaJa fGzlqx"
      >
        <div
          class="sc-htpNat wioCx"
        >
          <div
            class="sc-bxivhb joTsHV"
          >
            <h3
              class="sc-bwzfXH kgEpTV"
            >
              En ny digital tjänst
            </h3>
          </div>
          <div
            class="sc-EHOje hJrdUN"
          >
            <div
              class="Paragraph__StyledParagraph-n6tulf-0 kcTDfe"
            >
              <div>
                <p>
                  Aeneanurutrum, erat at tempus porttitor, mi eros vestibulum magna, eget semper mauris ipsum a nisi. Interdum et malesuada fames ac ante ipsum primis in faucibus. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Cras quis odio vel turpis pellentesque finibus. Pellentesque cursus diam ac ligula placerat volutpat. Vivamus dictum augue eu rutrum efficitur. Suspendisse ut pellentesque ante. Suspendisse odio velit, fringilla eu porta vel, elementum ac libero. Nam eu vestibulum magna. Vestibulum quis risus elementum, lobortis sapien at, rhoncus enim. Nam accumsan odio lacus, at facilisis lacus rhoncus vel. Suspendisse sollicitudin a ipsum at cursus.
                </p>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="react-reveal"
      style="opacity: 1;"
    >
      <div
        data-test="block-"
        class="sc-bdVaJa fGzlqx"
      >
        <div
          class="sc-htpNat wioCx"
        >
          <div
            class="sc-bxivhb joTsHV"
          >
            <h3
              class="sc-bwzfXH kgEpTV"
            >
              Process
            </h3>
          </div>
          <div
            class="sc-EHOje hJrdUN"
          >
            <div
              class="Paragraph__StyledParagraph-n6tulf-0 kcTDfe"
            >
              <div>
                <p>
                  Aeneanurutrum, erat at tempus porttitor, mi eros vestibulum magna, eget semper mauris ipsum a nisi. Interdum et malesuada fames ac ante ipsum primis in faucibus. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Cras quis odio vel turpis pellentesque finibus. Pellentesque cursus diam ac ligula placerat volutpat. Vivamus dictum augue eu rutrum efficitur. Suspendisse ut pellentesque ante. Suspendisse odio velit, fringilla eu porta vel, elementum ac libero. Nam eu vestibulum magna. Vestibulum quis risus elementum, lobortis sapien at, rhoncus enim. Nam accumsan odio lacus, at facilisis lacus rhoncus vel. Suspendisse sollicitudin a ipsum at cursus.
                </p>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      data-test="block-"
      class="GridColumn__GridColumnClean-sc-1xz61ww-0 sc-kAzzGY hwdyEL"
    >
      <div
        class="sc-bdVaJa fGzlqx"
      >
        <div
          class="Quote__ActualQuote-sc-1t3qoo6-0 caUjjr"
        >
          ""We feel that our crew at Iteam are more colleagues than suppliers. Rather than specifying a tech spec to work on, we collaborate, carefully chiseling out functionality as we go along. This way of working has minimized waste and it has also taken us further than we ever imagined”"
        </div>
        <div
          class="Quote__Person-sc-1t3qoo6-1 gOYqcc"
        >
          Anna Andersson, chef Arbetsförmedlingen
        </div>
      </div>
    </div>
    <div
      class="react-reveal"
      style="opacity: 1;"
    >
      <div
        data-test="block-"
        class="sc-bdVaJa fGzlqx"
      >
        <div
          class="sc-htpNat wioCx"
        >
          <div
            class="sc-bxivhb joTsHV"
          >
            <h3
              class="sc-bwzfXH kgEpTV"
            >
              Utveckling
            </h3>
          </div>
          <div
            class="sc-EHOje hJrdUN"
          >
            <div
              class="Paragraph__StyledParagraph-n6tulf-0 kcTDfe"
            >
              <div>
                <p>
                  Aeneanurutrum, erat at tempus porttitor, mi eros vestibulum magna, eget semper mauris ipsum a nisi. Interdum et malesuada fames ac ante ipsum primis in faucibus. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Cras quis odio vel turpis pellentesque finibus. Pellentesque cursus diam ac ligula placerat volutpat. Vivamus dictum augue eu rutrum efficitur. Suspendisse ut pellentesque ante. Suspendisse odio velit, fringilla eu porta vel, elementum ac libero. Nam eu vestibulum magna. Vestibulum quis risus elementum, lobortis sapien at, rhoncus enim. Nam accumsan odio lacus, at facilisis lacus rhoncus vel. Suspendisse sollicitudin a ipsum at cursus.
                </p>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="react-reveal"
      style="opacity: 1;"
    >
      <div
        data-test="block-"
        class="sc-bdVaJa fGzlqx"
      >
        <div
          class="sc-htpNat wioCx"
        >
          <div
            class="sc-bxivhb joTsHV"
          >
            <h3
              class="sc-bwzfXH kgEpTV"
            >
              Arbetsförmedlingen
            </h3>
          </div>
          <div
            class="sc-EHOje hJrdUN"
          >
            <div
              class="Paragraph__StyledParagraph-n6tulf-0 kcTDfe"
            >
              <div>
                <p>
                  Aeneanurutrum, erat at tempus porttitor, mi eros vestibulum magna, eget semper mauris ipsum a nisi. Interdum et malesuada fames ac ante ipsum primis in faucibus. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Cras quis odio vel turpis pellentesque finibus. Pellentesque cursus diam ac ligula placerat volutpat. Vivamus dictum augue eu rutrum efficitur. Suspendisse ut pellentesque ante. Suspendisse odio velit, fringilla eu porta vel, elementum ac libero. Nam eu vestibulum magna. Vestibulum quis risus elementum, lobortis sapien at, rhoncus enim. Nam accumsan odio lacus, at facilisis lacus rhoncus vel. Suspendisse sollicitudin a ipsum at cursus.
                </p>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="react-reveal"
      style="opacity: 1;"
    >
      <div
        data-test="block-"
        class="sc-bdVaJa fGzlqx"
      >
        <div
          class="sc-htpNat wioCx"
        >
          <div
            class="sc-bxivhb joTsHV"
          >
            <h3
              class="sc-bwzfXH kgEpTV"
            >
              Samarbetspartners
            </h3>
          </div>
          <div
            class="sc-EHOje hJrdUN"
          >
            <div
              class="Paragraph__StyledParagraph-n6tulf-0 kcTDfe"
            >
              <div>
                <p>
                  <a
                    href="https://hej.com"
                  >
                    Digitalbyrån
                  </a>
                </p>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="react-reveal"
      style="opacity: 1;"
    >
      <div
        data-test="block-"
        class="sc-bdVaJa fGzlqx"
      >
        <div
          class="sc-htpNat wioCx"
        >
          <div
            class="sc-bxivhb joTsHV"
          >
            <h3
              class="sc-bwzfXH kgEpTV"
            >
              Presskontakt
            </h3>
          </div>
          <div
            class="sc-EHOje hJrdUN"
          >
            <div
              class="Paragraph__StyledParagraph-n6tulf-0 kcTDfe"
            >
              <div>
                <ul>
                  <li>
                    <a
                      href="mailto:hans.rollman@iteam.se"
                    >
                      Hans Rollman
                    </a>
                  </li>
                  <li>
                    Johan Littorin, Telenor, 0734 41 72 85
                  </li>
                </ul>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      data-test="block-"
      class="GridColumn__GridColumnClean-sc-1xz61ww-0 sc-cSHVUG eWEIce"
    >
      <div
        class="sc-bdVaJa fGzlqx"
      >
        <h3
          class="sc-bwzfXH kgEpTV"
        >
          Tekniker vi använt
        </h3>
        <div
          class="Frameworks__FrameworksWrap-qcim0a-0 cTevWf"
        >
          <div
            class="Frameworks__Framework-qcim0a-1 gKQzeM"
          >
            <img
              src="https://images.ctfassets.net/rj4r6yfcesw5/3ipy8GBepacEyeKM0weI0i/39509f189b3f80400357320c6def15ad/docker.png"
              class="Frameworks__FrameworkIcon-qcim0a-2 eJpXWR"
            />
            Docker
          </div>
          <div
            class="Frameworks__Framework-qcim0a-1 gKQzeM"
          >
            <img
              src="https://images.ctfassets.net/rj4r6yfcesw5/6Khn8L48laAoQuCkaouyQG/08e4cf1fe02bd945c23807d5a8f29e87/react.png"
              class="Frameworks__FrameworkIcon-qcim0a-2 eJpXWR"
            />
            React
          </div>
          <div
            class="Frameworks__Framework-qcim0a-1 gKQzeM"
          >
            <img
              src="https://images.ctfassets.net/rj4r6yfcesw5/5EBgyxee0oiGqqOmAEmAUq/af8d1ae7d3d578197205205b2c33e8ff/Vue.png"
              class="Frameworks__FrameworkIcon-qcim0a-2 eJpXWR"
            />
            Vue
          </div>
          <div
            class="Frameworks__Framework-qcim0a-1 gKQzeM"
          >
            <img
              src="https://images.ctfassets.net/rj4r6yfcesw5/3CIQ81A1M4EueKkYA8YgMY/ff59095490dd17d1ab74282447e0a213/nodejs.png"
              class="Frameworks__FrameworkIcon-qcim0a-2 eJpXWR"
            />
            NodeJS
          </div>
          <div
            class="Frameworks__Framework-qcim0a-1 gKQzeM"
          >
            <img
              src="https://images.ctfassets.net/rj4r6yfcesw5/4RmR16im884ykaUgEaiscA/411607987ba022b96e9a29d000821aef/mandrill.png"
              class="Frameworks__FrameworkIcon-qcim0a-2 eJpXWR"
            />
            Mandrill
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
